chall01/vgrankul.py

```diff
@@ -0,0 +1,50 @@
+#!/bin/python3
+import sys
+
+sos = [
+".-",
+"-...", 
+"-.-.", 
+"-..", 
+".", 
+"..-.", 
+"--.", 
+"....", 
+"..", 
+".---", 
+"-.-", 
+".-..", 
+"--", 
+"-.", 
+"---", 
+".--.", 
+"--.-", 
+".-.", 
+"...", 
+"-", 
+"..-", 
+"...-", 
+".--", 
+"-..-", 
+"-.--", 
+"--.."]
+
+i = 0
+letter = ['a', 'b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
+Bletter = ['A', 'B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']
+sos_string = ""
+
+if len(sys.argv) != 2:
+    print("usage: ./vgrankul.py <a-zA-Z string>")
+else:
+    string = sys.argv[1]
```

Comment:
you could have used `.lower()` to sys.argv[1] before processing to avoid having to handle both and having 2 arrays